src/ape_cache/_cli.py

```diff
@@ -0,0 +1,40 @@
+import click
+import pandas as pd
+
+from ape.cli import NetworkBoundCommand, network_option
+from ape.utils import ManagerAccessMixin, logger  # type: ignore
+
+
+def get_engine():
+    return ManagerAccessMixin.query_manager.engines["cache"]
+
+
+@click.group(short_help="Query from caching database")
+def cli():
+    """
+    Command-line helper for managing query caching database.
+    """
+
+
+@cli.command(cls=NetworkBoundCommand, short_help="Initialize a new cache database")
+@network_option()
+def init(network):
+    get_engine().init_db()
+    logger.info("Caching database initialized.")
+
+
+@cli.command(
+    cls=NetworkBoundCommand, short_help="Call and print SQL statement to the cache database"
+)
+@network_option()
+@click.argument("sql")
```

Comment:
It might also be nice to read from a file. That way, users can save `.sql` files and they are like saved-searches then

Reply:
Good suggestion!

Reply:
I have not implemented this, and I'm not really sure what you mean here. Do you mean save SQL queries in our hidden folder that can be called?

Reply:
Ya (a future idea - we can make a ticket). Maybe we have a `queries/` directory and then files in there named `gas_stats.sql` or something and then the user can do:

```bash
ape query gas_stats
```

and it runs the saved search

Reply:
The idea is that it should be much more ergonomic to do it through our classes (e.g. `chain.blocks.querty(...)`), but not opposed to this